tests/robustness/coverage/README.md

```diff
@@ -0,0 +1,123 @@
+## Overview
+
+This is still Work In Progress.
+
+Go tests in this directory analyze the usage of Etcd API based on collected
```

Comment:
This is doesn't provide enough context, why, what and how. The test is to track coverage of K8s-etcd contract, but the readme doesn't mention it at all.